packages/qwik/src/core/v2/shared/scheduler.ts

```diff
@@ -76,6 +76,7 @@ export const createScheduler = (container: Container2, scheduleDrain: () => void
     $scheduleTask$: scheduleTask,
     $scheduleComputed$: scheduleComputed,
     $scheduleNodeDiff$: scheduleNodeDiff,
+    $scheduleNodeProp$: scheduleNodeProp,
```

Comment:
I think this whole file can be reverted. See comment about scheduler not needed.